src/fetch/data_preprocessing.py

```diff
@@ -0,0 +1,43 @@
+"""Preprocessing of data for payments
+The processing in this file should be minimal, mostly to correct for incorrect price data or to
+exclude trades from accounting.
+Any processing which is not required anymore should be removed.
+If this file contains any processing, the accounting period should be added in the description,
+with a short description of the processing."""
+
+from pandas import DataFrame
+
+
+def preprocess_batch_data(batch_data_df: DataFrame) -> DataFrame:
+    """Preprocess batch data"""
+    return batch_data_df
+
+
+def preprocess_trade_data(trade_data_df: DataFrame) -> DataFrame:
+    """Preprocess trade data
+    Accounting period starting on: 2024-05-07
+    This processing corrects prices of FXN token which was off until the middle of the
+    accounting period. The price 35322822331170980 was the last price before it became
+    unreasonably large (due to an out of sync $0.02 TVL Uniswap pool).
+    """
+    trade_data_df.loc[
+        (trade_data_df["buy_token"] == "0x365accfca291e7d3914637abf1f7635db165bb09")
```

Comment:
I would suggest that we also check the auction range, so that we restrict the fix to the auctions that were affected 2 weeks ago.

The reason is that in case this happens again, maybe the price needs to be set to something different.

Reply:
My preference would be to remove any modification once it is not needed anymore. If version used for payments are tagged appropriately (e.g. `2024-05-21` for the payouts executed on that day), one can get to the exact version used for a payment.